cumulus_library/template_sql/sql_utils.py

```diff
@@ -26,8 +26,8 @@
 
 
 @dataclass(kw_only=True)
-class BaseConfig(abc.ABC):
-    """Abstract ase class for handling table detection/denormalization"""
+class BaseConfig:
+    """Base class for handling table detection/denormalization"""
```

Comment:
ruff complained about this - there are no abstract methods or properties now that this is a simple dataclass.